katsdpcontroller/generator.py

```diff
@@ -1718,29 +1737,31 @@ def _render_continuum_parameters(parameters):
     return '; '.join('{}={!r}'.format(key, value) for key, value in parameters.items())
 
 
-def _make_continuum_imager(g, config, capture_block_id, name, telstate, target_cache):
+def _make_continuum_imager(g, config, capture_block_id, name, telstate, target_mapper):
     output = config['outputs'][name]
     l1_flags_name = output['src_streams'][0]
     l0_name = config['outputs'][l1_flags_name]['src_streams'][0]
     l0_info = L0Info(config, l0_name)
     l0_stream = name + '.' + l0_name
     data_url = _stream_url(capture_block_id, l0_stream)
     cpus = _continuum_imager_cpus(config)
-    targets = _unique_targets(config, capture_block_id, name, telstate, target_cache)
+    # Image targets observed for at least 15 minutes
+    targets = _get_targets(config, capture_block_id, name, telstate, 15 * 60)
```

Comment:
Fair enough, will do.

I'm also thinking it might be nice (mainly for development) to let them be overridden from config - mostly for development where I don't want to have to do a 2 hour simulation just to check the spectral imager plumbing.